### DISCUSSION TITLE
Create and push tag `v_2.0.7`

### DISCUSSION THREAD

**Discussion by rickie** [qos-ch/slf4j#341] [General]
The [release notes](https://www.slf4j.org/news.html) contain the following line: 

> The binary of this version can be reproduced by checking out the tag v_2.0.7 from the source code repository (GitHub)

However, the tag doesn't exist yet on GitHub: https://github.com/qos-ch/slf4j/tree/v_2.0.7.
As a result, comparing the release against the previous tag doesn't work: https://github.com/qos-ch/slf4j/compare/v_2.0.6...v_2.0.7.

Can someone take a look at this 😄? 

(I'm not sure if this is the best place to create this, but the "Issues" tab isn't available in this repository.)

**Comment by ceki** [comment]
@rickie Thank you for this report. Issues can be reported at [jira.qos.ch](https://jira.qos.ch) 

Anyway, the missing tag has been added.

**Comment by rickie** [comment]
Thanks for pushing the tag 😄. Next time I'll use the Jira way.
